easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'FLASH'
+version = '1.2.11'
+
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software
+  tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to
+  merge pairs of reads when the original DNA fragments are shorter than twice the length of reads.
+  The resulting longer reads can significantly improve genome assemblies. They can also improve
+  transcriptome assembly when FLASH is used to merge RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['http://download.sourceforge.net/%(namelower)spage']
```

Comment:
Ah, ok, sorry, I was already wondering what the `page` thing was doing there, I should have realised. :)